cdap-examples/FileSetExample/src/main/java/co/cask/cdap/examples/fileset/FileSetService.java

```diff
@@ -106,6 +109,7 @@ public void read(HttpServiceRequest request, HttpServiceResponder responder,
      */
     @PUT
     @Path("{fileset}")
+    @TransactionPolicy(TransactionControl.EXPLICIT)
```

Comment:
They don't have to. But they are an example for how you can use explicit tx control.